packages/flutter/lib/src/material/drawer_item.dart

```diff
@@ -26,8 +36,13 @@ class DrawerItem extends StatelessWidget {
   /// The widget below this widget in the tree.
   final Widget child;
 
+  /// Callend when the user taps this drawer item.
   final VoidCallback onPressed;
 
+  /// Whether this drawer item is currently selected.
+  ///
+  /// The currently selected item is highlighted with additional color to
```

Comment:
Ok.  Yeah, the default is sort of a gray and the selected one is colorful.  Just "highlighted" is probably more accurate because I think we draw a background too.